src/main/views/macro/support.njk

```diff
@@ -1,20 +1,41 @@
 {% from "externalLink.njk" import externalLink %}
 
+{% macro _webChatWidget(label = 'Web chat') %}
+      <h2 class="heading-medium" style="margin-bottom:5px; margin-top: 10px;">{{ t(label)}}</h2>
+      <div id="ctsc-web-chat"></div>
+{% endmacro%}
+
+{% macro _emailWidget(email = 'moneyclaims@justice.gov.uk') %}
```

Comment:
this email should be retrieved from the CONTACT_EMAIL variable

Reply:
done